frontend/public/locales/en/translation.json

```diff
@@ -1288,6 +1292,7 @@
   "Ensure these settings are correct. The saved cluster draft will be used to determine the available network resources. Therefore after you press Save you will not be able to change these cluster settings.": "Ensure these settings are correct. The saved cluster draft will be used to determine the available network resources. Therefore after you press Save you will not be able to change these cluster settings.",
   "Enter a name for this search query": "Enter a name for this search query",
   "Enter a value": "Enter a value",
+  "Enter additional networks in the format <namespace>/<name>": "Enter additional networks in the format <namespace>/<name>",
```

Comment:
Can we make this singular? Within the text field, you can only enter 1 network name.